mailbox/event/json/src/main/scala/org/apache/james/event/json/EventSerializer.scala

```diff
@@ -26,15 +26,13 @@ import julienrf.json.derived
 import org.apache.james.core.quota.{QuotaCount, QuotaSize, QuotaValue}
 import org.apache.james.core.{Domain, User}
 import org.apache.james.mailbox.MailboxListener.{QuotaEvent => JavaQuotaEvent, QuotaUsageUpdatedEvent => JavaQuotaUsageUpdatedEvent}
-import org.apache.james.mailbox.model.{QuotaRoot, Quota => JavaQuota}
-import play.api.libs.json.{JsError, JsNull, JsNumber, JsObject, JsResult, JsString, JsSuccess, Json, OFormat, Reads, Writes}
+import org.apache.james.mailbox.model.{MailboxId, QuotaRoot, Quota => JavaQuota}
+import org.apache.james.mailbox.{Event => JavaEvent}
+import play.api.libs.json._
 
 import scala.collection.JavaConverters._
-import scala.compat.java8.OptionConverters
-
-private sealed trait QuotaEvent {
-  def getQuotaRoot: QuotaRoot
 
+private sealed trait Event {
```

Comment:
the below line should also being refactored, WDYT?
```scala
def toJava: JavaQuotaEvent
```

Reply:
For the record I did it in a later not yet PRe commit.